CONTRIBUTING.md

```diff
@@ -1,52 +1,72 @@
+* For contributing to CI and e2e, refer to this [document](./contribute/CONTRIBUTING-TO-CI-AND-E2E.md)
+	- Refer to this [overview](./e2e/README.md) of e2e.
+	
+Refer to this [document](./contribute/design/code-structuring.md) for more information on code structuring and guidelines to follow on the same.
 
-Provide PRs for bug fixes or enhancements.
+## Solve Existing Issues
+Head over to [issues](https://github.com/openebs/openebs/issues) to find issues where help is needed from contributors. See our [list of labels guide](./contribute/labels-of-issues.md) to help you find issues that you can solve faster.
 
+A person looking to contribute can take up an issue by claiming it as a comment/assign their Github ID to it. In case there is no PR or update on progress for a week on the said issue, then the issue opens up again, for anyone to take up again. We need to consider high priority issues/regressions where response time needs to be a day or so. 
+
+---
 ### Sign your work
 
-We use the Developer Certificate of Origin (DCO) as a additional safeguard
-for the OpenEBS project. This is a well established and widely used
-mechanism to assure contributors have confirmed their right to license
-their contribution under the project's license.
-Please read [developer-certificate-of-origin](https://github.com/openebs/openebs/blob/master/contribute/developer-certificate-of-origin).
+We use the Developer Certificate of Origin (DCO) as an additional safeguard for the OpenEBS project. This is a well established and widely used mechanism to assure contributors have confirmed their right to license their contribution under the project's license. Please read [developer-certificate-of-origin](./contribute/developer-certificate-of-origin.md).
+
 If you can certify it, then just add a line to every git commit message:
 
 ````
   Signed-off-by: Random J Developer <random@developer.example.org>
 ````
+or use the command `git commit -s -m "commit message comes here"` to sign-off on your commits.
+
+Use your real name (sorry, no pseudonyms or anonymous contributions). If you set your `user.name` and `user.email` git configs, you can sign your commit automatically with `git commit -s`. You can also use git [aliases](https://git-scm.com/book/tr/v2/Git-Basics-Git-Aliases) like `git config --global alias.ci 'commit -s'`. Now you can commit with `git ci` and the commit will be signed.
+
+---
+
+## Join our community 
 
-Use your real name (sorry, no pseudonyms or anonymous contributions).
-If you set your `user.name` and `user.email` git configs, you can sign your
-commit automatically with `git commit -s`. You can also use git [aliases](https://git-scm.com/book/tr/v2/Git-Basics-Git-Aliases)
-like `git config --global alias.ci 'commit -s'`. Now you can commit with
-`git ci` and the commit will be signed.
+Want to actively develop and contribute in the OpenEBS community, refer to this [document](./community/README.md).
```

Comment:
end the line with a ? mark.

Reply:
That would be a rhetorical question. Hence I think quite right the way it is. Suggestions.
Even if there would be a question mark then it could be
`Want to actively develop and contribute in the OpenEBS community? refer to this [document].`